app/scripts/components/navigatorComponents/listItems.jsx

```diff
@@ -239,9 +238,11 @@ class ListItems extends React.Component {
     tableRowColumnMenu(child, menuWidth, projectRoles) {
         let optionsMenu
         if (child.kind === Kind.DDS_FILE) {
-            optionsMenu = <FileOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FILE, true)}/>
+            let role = projectRoles.get(child.project.id);
+            optionsMenu = role && role === 'Project Admin' && <FileOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FILE, true)}/>
```

Comment:
This will only show the file options menu if the user is a project admin. They should also be able to see the menu no matter what permissions they have. Even a project_viewer can view and add provenance.

Reply:
so, FileOptionsMenu should always be shown with a file, i will roll back those changes